### PR TITLE
refs #132 fix popup shift when avatar appears late

### PR DESCRIPTION
**Pull request by eneiluj**
It should solve the problem mentioned in #132 for all browsers. If the image size is set before it has been loaded, popup size does not change after having been displayed and the shift does not happen.

### CODE REVIEW

**Comment by anaqreon** [comment]
I can confirm that this pull request does fix the issue in the Chromium web browser (which is not a problem in Firefox). The popup position is centered on the contact marker image; it is not coincident with the contact marker as of commit 4439fd5d3d07a7dfc34f0570134fb95b83c0620c.

**Comment by jancborchardt** [comment]
Added a commit which fixes the popup being too big for contacts with a profile image (cause it just sized it up to whatever size the contact image had, since the width limit was missing). :) Now it works in both Chrome and Firefox.

Anything about the js/contactsController.js conflict? Will see if I can resole it.

**Comment by jancborchardt** [comment]
@anaqreon I did the conflict resolution, but now for some reason your change with moving the popover doesn’t work anymore, even though as far as I can see I took it over 1 to 1. Sorry – do you have any idea what’s wrong there?

**Comment by jancborchardt** [comment]
@eneiluj @tacruc the line that @anaqreon changed was removed completely in master, so I assume something changed regarding how to position the popup?

**Comment by eneiluj** [comment]
As far as I can see, @anaqreon's commit doesn't have any effect because contact tooltip shows/hides thanks to `onContactMouseover` and `onContactMouseout` methods. There should be no tooltip bind. It's done that way to avoid showing the tooltip when the popup is there.

I guess @anaqreon didn't pull fix-issue132 before working or I don't quite understand what happened.

Anyway, I feel we're mixing two topics here.

* wrong popup/tootlip placement when avatar appears too late or is too big (fixed by @jancborchardt and @eneiluj). It was because the tooltip/popup size was changing after it was displayed, making it move.
* choose whether contact tooltips/popups should be placed at the exact marker place or at its center. Personally I prefer showing it at the center to better express the fact it's related to the contact marker which stays visible. Otherwise tooltip/popup hide the marker and it feel weird.

Bottom line: I think we could keep @jancborchardt's commit and get rid of @anaqreon's, then merge this PR, then ask ourselves where to place tooltips/popups. What do y'all think?

**Comment by eneiluj** [comment]
To make it clearer, I removed @anaqreon's commit and pushed again `fix-issue132`. It looks good to merge to me.

**Comment by anaqreon** [comment]
Wow, I have no idea how I screwed up that simple change so royally. Sorry guys.

**Comment by eneiluj** [comment]
@anaqreon Don't worry, I'm pretty sure everybody messed up with git at least a thousand times :wink:.

**Comment by jancborchardt** [comment]
> Bottom line: I think we could keep @jancborchardt's commit and get rid of @anaqreon's, then merge this PR, then ask ourselves where to place tooltips/popups. What do y'all think?

Sounds good! :)

And as said in the other issue, I do think @anaqreon’s change was good, as it just looks like the marker is growing. (Whereas currently it’s a marker on a marker, which is weird. :D)

**Comment by jancborchardt** [comment]
@anaqreon so if you want to open another pull request for the position of the popover after click, that’d be cool! :)
